### ISSUE TITLE
Error throwed in Chrome console after upgrade to 4

### ISSUE THREAD

**Issue by pd4d10** [AliasIO/Wappalyzer#1487]
~~Uncaught ReferenceError: regeneratorRuntime is not defined
    at mobile-c02f111….js:3
    at mobile-c02f111….js:3
    at mobile-c02f111….js:1~~

See @malimichael 's reply

**Comment by malimichael** [comment]
I'm also having an issues with v4

```
browser-polyfill.js:590 Uncaught (in promise) 
Object {message: "The message port closed before a response was received."}
```

**Comment by pd4d10** [comment]
Sorry for my mistake, error throwed from `mobile-c02f111….js` has nothing to do with this extension. It's a JS file of website itself.

See @malimichael's reply, same in my browser.

**Comment by leadream** [comment]
I also have this problem.

**Comment by ludwigheymbeeck** [comment]
confirmed here as well

**Comment by zhaozhiming** [comment]
same to me

**Comment by AminulBD** [comment]
Same problem here.

**Comment by MoeinAlizadeh** [comment]
+ same !

**Comment by vinioliveira** [comment]
Same thing happened to me

**Comment by thanhquang2608** [comment]
Same to me

**Comment by guanxinHETIC** [comment]
+1

**Comment by zhonghuiping** [comment]
same to me

**Comment by themazim** [comment]
+1

**Comment by amandaluu** [comment]
+1

**Comment by mitrofun** [comment]
+1

**Comment by jpmartinezv** [comment]
Same to me

**Comment by jeeteshnariya** [comment]
yaa it's is disable wappalyzer ..! @MannyC

**Comment by salmanaligeek** [comment]
See this http://imgur.com/LAKGxdL

**Comment by emaxedon** [comment]
+1

**Comment by DaAwesomeP** [comment]
This is from the new WebExtension version of the addon. To be clear, are you seeing this error and the extension is *not* functioning, or are you seeing this error and the extension is still functioning as normal?

**Comment by ssosina** [comment]
+1

**Comment by serjo96** [comment]
same problem

**Comment by antwal** [comment]
same after upgrade

**Comment by aalencar** [comment]
same problem here

**Comment by develpr** [comment]
+1

**Comment by AliasIO** [comment]
It's fixed in v4.0.1.
